### PR TITLE
Lerna link convert

### PR DESCRIPTION
**Pull request by mikhailrojo**
see [lerna doc about common deps](https://github.com/lerna/lerna#common-devdependencies)

I just make `lerna link convert` - so all dependencies got hoisted to the root package.json
It will release  cluttered individual packages from the common devDeps

### CODE REVIEW

File: packages/zipkin-encoder-thrift/package.json [hunk 1]
```diff
@@ -22,6 +22,6 @@
   "homepage": "https://github.com/openzipkin/zipkin-js#readme",
   "dependencies": {
     "thrift": "^0.11.0",
-    "zipkin": "^0.21.0"
+    "zipkin": "file:../zipkin"
```
Comment (jcchavezs) [inline packages/zipkin-encoder-thrift/package.json:25]: I see this link https://github.com/lerna/lerna#common-devdependencies specifically talks about `devDependencies` but this change changes `dependencies`, I am curious about what would be the version being pinned for this.

Comment (mikhailrojo) [reply packages/zipkin-encoder-thrift/package.json:25]: the current version in packages/zipkin
currently 0.21.0, but when upgraded 0.22.0

as lerna here works as in fixed(not independent mode) every time we change in one lib, every other upgrades automatically.

**Comment by mikhailrojo** [comment]
I just realized that dependencies are managed by yarn workspaces - and this PR is useless
stupid me

**Comment by jcchavezs** [comment]
We still appreciate the effort. Thanks a lot!

**Pull request closed without merging**